### PR TITLE
Fix: mrope_position_delta missing in PD disaggregation decode mode for qwen-vl series

### PR DESCRIPTION
**Pull request by LeoGoTi**
This PR has been closed.
Check for [this PR](https://github.com/sgl-project/sglang/pull/8094#issue-3235682446) as it's handling this issue.

## Motivation
This PR aims to fix a bug where `mrope_position_delta` is missing on PD-disaggregation setup, for decode mode only.

In brief:
- Qwen-vl series models implement the mrope algorithm stage one for `mrope_positions` and `mrope_position_delta`.
- `mrope_position_delta` is necessary for generating subsequent positions during the decode stage, and is not accessible when PD disaggregation is deployed.

## Modifications
- Add fields named `decode_mrope_position_delta` to the `Req` and `ModelWorkerBatch` dataclasses.
- On `_compute_mrope_positions` calculation, if the delta values are available from the `decode_mrope_position_delta` field, we use it directly.
- Add tensor allocation named `mrope_position_delta` in `MetaDataBuffers`, serving as a kv-meta part being transferred from prefill to decode. This is only valid when we register `MetaDataBuffers` using `is_mrope_enabled=True`.

## Checklist

- [x] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

File: python/sglang/srt/disaggregation/utils.py [hunk 1]
```diff
@@ -126,6 +128,10 @@ def __init__(
             self.output_hidden_states = torch.zeros(
                 (size, hidden_size), dtype=dtype, device=device
             )
+            if is_mrope_enabled:
+                self.mrope_position_delta = torch.zeros(
+                    (size, 16), dtype=torch.int32, device=device
+                )
```
[the remark this answers is not in the document]
Comment (LeoGoTi) [reply python/sglang/srt/disaggregation/utils.py:134]: The size of the second dimension is set to `16` also to pad a single meta for delta value to 64B, although only the first element is used

**Comment by LeoGoTi** [comment]
Closed this PR as [another active PR](https://github.com/sgl-project/sglang/pull/8094#issue-3235682446) is doing this more elegantly.
